src/predefined/continuous_famous_systems.jl

```diff
@@ -1405,3 +1498,297 @@ end
     du2 = 1/τ*(-N + G)
     return SVector{2}(du1, du2)
 end
+
+"""
+```julia
+sakarya(u0= [-2.8976045, 3.8877978, 3.07465];
+    a =-1.0,
+    b = 1.0,
+    c = 1.0,
+    h = 1.0,
+    p = 1.0,
+    q = 0.4,
+    r = 0.3,
+    s = 1.0
+)
+```
+```math
+\\begin{aligned}
+\\dot{x} &= ax + hy + syz\\\\
+\\dot{y} &= -by - px + qxz \\\\
+\\dot{z} &= cz - rxy
+\\end{aligned}
+```
+A system presenting robust chaos that varies from single wing to double wings to four wings.
+Its attractor arises due to merging of two disjoint bistable attractors [^Li2015].
+
+[^Li2015]:
+    Li, Chunbiao, et al (2015). A novel four-wing strange attractor born in bistability.
+    IEICE Electronics Express 12.4.
+"""
+function sakarya(u0= [-2.8976045, 3.8877978, 3.07465];
+    a =-1.0,
+    b = 1.0,
+    c = 1.0,
+    h = 1.0,
+    p = 1.0,
+    q = 0.4,
+    r = 0.3,
+    s = 1.0
+)
+    return CDS(loop_sakarya, u0, [a,b,c,h,p,q,r,s])
+end
+
+function loop_sakarya(u, p, t)
```

Comment:
`loop_sakarya` doesn't make seense. `loop` meant `lorenz out of place`. You should just use `sakarya_rule` as the name of the function.